app/imports/templates/components/nameStatus.js

```diff
@@ -216,19 +216,25 @@ Template['status-reveal'].helpers({
 
 Template['aside-reveal'].helpers({ 
   registrationDate() {
+    if (Template.instance().data == null) return;
```

Comment:
basically they were returning template exceptions on the console, before data was loaded, this is to prevent them from doing so